raster-foundry/raster-foundry
# Display user info on template items

## Overview

This PR updates user info on template items so that it displays actual user name/email/id when template owner is not `default`.

### Checklist


## Testing Instructions

 * Go to the list of templates and check if `Created by` displays user info when a templates is created by a user, or displays platform name when a template is created by the default user.

Closes #4045
